blocks.mm

```diff
@@ -26,7 +27,7 @@ void logBlock(FILE *file, id block) {
   if (signature) {
     NSMethodSignature *methodSignature = [NSMethodSignature signatureWithObjCTypes:signature];
     Class kind = object_getClass(block);
-    fprintf(file, "<%s@%p signature=\"%s ; retType=%s", class_getName(kind), (void *)block, signature, methodSignature.methodReturnType);
+    fprintf(file, "<%s@%p signature=\"%s ; retType=%s", class_getName(kind), (__bridge void *)block, signature, methodSignature.methodReturnType);
```

Comment:
Does this work properly in ARC and non-ARC environments?

Reply:
Yes, this satisfies Xcode with ARC enabled and Theos without it.

Reply:
Gotcha, thanks